project-fortis-interfaces/src/components/Insights/Maps/HeatMap.js

```diff
@@ -77,8 +78,12 @@ export default class HeatMap extends React.Component {
   componentWillReceiveProps(nextProps) {
     const { placeid, defaultZoom } = this.state;
     const { targetBbox } = this.props;
-
-    if (hasChanged(this.props, nextProps) && nextProps.selectedplace.placeid && placeid !== nextProps.selectedplace.placeid) {
+    const { dashboardIsLoadedFromShareLink } = nextProps;
+    if (hasChanged(this.props, nextProps) && dashboardIsLoadedFromShareLink && !this.state.sharedLinkMapRepositions) {
+      this.refs.map.leafletElement.fitBounds([this.getMapBounds(nextProps.bbox)][0]);
```

Comment:
Can this be rationalized with the `moveMapToBoundingBox` method defined below?

https://github.com/CatalystCode/project-fortis/blob/426df52ff19f8efcd4468f30c0553267f53683b9/project-fortis-interfaces/src/components/Insights/Maps/HeatMap.js#L104-L108